src/themes/playful.css

```
@@ -334,14 +350,415 @@
+      --background: var(--wa-color-success-fill-subtle);
+      --content-color: var(--wa-color-success-text-on-fill);
+    }
+    &[variant='warning'] {
+      --border-color: var(--wa-color-warning-border-subtle);
+      --background: var(--wa-color-warning-fill-subtle);
+      --content-color: var(--wa-color-warning-text-on-fill);
+    }
+    &[variant='danger'] {
+      --border-color: var(--wa-color-danger-border-subtle);
+      --background: var(--wa-color-danger-fill-subtle);
+      --content-color: var(--wa-color-danger-text-on-fill);
+    }
+    &[variant='neutral'] {
+      --border-color: var(--wa-color-neutral-border-subtle);
+      --background: var(--wa-color-neutral-fill-subtle);
+      --content-color: var(--wa-color-neutral-text-on-fill);
+    }
+  }
+
+  wa-badge {
+    --border-radius: var(--wa-corners-l);
+    --border-width: 0;
+
+    &[variant='brand'] {
+      --background: var(--wa-color-brand-80);
+      --content-color: var(--wa-color-brand-20);
+    }
+    &[variant='success'] {
+      --background: var(--wa-color-green-80);
+      --content-color: var(--wa-color-green-20);
+    }
+    &[variant='warning'] {
+      --background: var(--wa-color-yellow-80);
+      --content-color: var(--wa-color-yellow-20);
+    }
+    &[variant='danger'] {
+      --background: var(--wa-color-red-80);
+      --content-color: var(--wa-color-red-20);
+    }
+    &[variant='neutral'] {
+      --background: white;
+      --content-color: var(--wa-color-base-30);
+    }
+  }
+
+  .preview-container {
+    overflow: hidden;
+  }
+
+  .project-header {
+    --wa-color-text-normal: var(--wa-color-success-text-on-fill);
+  }
+
+  .hero-background {
+    position: absolute;
+    z-index: -1;
+    top: -18%;
+    left: 50%;
+    width: 250rem;
+    height: 250rem;
+    transform: translate(-50%, -50%);
+    flex-shrink: 0;
+    border-radius: 50%;
+    background: linear-gradient(180deg, var(--wa-color-green-90) 69.42%, var(--wa-color-green-80) 100%);
+  }
+
+  .hero-background::after {
+    background: url(/assets/images/kitchen-sink/playful/hero.png) 0 50% no-repeat;
+    content: '';
+    position: absolute;
+    width: 520px;
+    height: 600px;
+    transform: translateX(-50%);
+    left: 50%;
+    top: 185rem;
+  }
+
+  .hero {
+    padding-top: 19rem;
+    text-align: center;
+  }
+
+  .hero .hero-title {
+    color: var(--wa-color-success-text-on-fill);
+    font-size: var(--wa-font-size-4xl);
+  }
+
+  .hero .hero-cta {
+    font-size: var(--wa-font-size-l);
+  }
+
+  .product-card::part(footer) {
+    border-width: 0;
+  }
+
+  .product-card wa-button {
+    margin-inline-end: var(--wa-space-xs);
+  }
+
+  .blog wa-avatar {
+    --size: 2rem;
+  }
+
+  .message-composer wa-card {
+    --background: var(--wa-color-surface-raised);
+    --wa-border-width-s: calc(var(--wa-border-width-base) * 1rem);
+  }
+
+  .message-composer wa-card::part(header) {
+    --wa-color-neutral-border-subtle: var(--wa-color-base-70);
```

```suggestion
    --wa-color-neutral-border-subtle: var(--wa-color-base-70);
    border-start-start-radius: calc(var(--border-radius) - var(--border-width));
    border-start-end-radius: calc(var(--border-radius) - var(--border-width));
```

We may want to subtract any borders from our border-radii to ensure we don't get artifacts peeking through (especially in dark mode).

---

Ooo nice!